irods_testing_environment/irods_setup.py

```diff
@@ -264,8 +264,19 @@ def build_input_for_catalog_consumer(self):
             '' # confirmation of inputs
         ]
 
+        # Handle the difference between 4.3 servers and 5.x servers.
```

Comment:
can we just make this use unattended_setup.json instead?

these array integers make me nervous/stabby

Reply:
Created #232